momentum-design/momentum-react-v2
# chore: install husky automatically

# Description

As per Husky 7 docs (https://github.com/typicode/husky/tree/v7.0.4/docs#yarn-2), yarn 2+ doesn't support the `prepare` script therefore there is a workaround.
- Run `husky install` when postinstall is run
- Since this package is public and published on npm, use pinst to disable the postinstall script when the package is being published.

# Testing

To test this PR:
1. Clone this branch to a new folder
2. Confirm that .git/config doesn't have core.hooksPath set
3. Run `yarn`
4. Confirm that .git/config now has core.hooksPath set to .husky
